howtos/setting_up_codacy_with_github.md

```diff
@@ -0,0 +1,94 @@
+# How to setup Codacy for an Eiffel project
+
+This walkthrough is based on Emelie Pettersons walkthrough on [Eiffel Intelligence issue #320](https://github.com/eiffel-community/eiffel-intelligence/pull/320)
+
+You login with you GitHub account. When you login Codacy will request for some permissions to your
+ GitHub account. This is to able to fetch all you public repositories into Codacy. For more
+ information take a look at their help page [Which permissions does Codacy need from my account?](https://support.codacy.com/hc/en-us/articles/115003405529-Which-permissions-does-Codacy-need-from-my-account-)
+
+After you have logged in you will need to import project to work with. If this is the first time
+ it is recommended to start by importing your own fork to try out. You will however need to import
+ the real project later on.
+
+When you have imported a project, Codacy will put an notification on each pull request in the
+ status part. If you want to have more interaction checkout the *Integrations* tab under the
+ Setting page. In GitHub you can find this integration in *Webhooks* tab under the Settings page.
+
+At this point you have integrated Codacy into GitHub so it is performing style checks for your
+ project on each Pull Request as this it the default behavior for the Webhook.
+
+For here on it is assumed we are working the project and not the fork
```

Comment:
'From here on' and maybe 'on the project'?

Reply:
Done